doc/userguide/rules/intro.rst

```diff
@@ -276,6 +276,7 @@ Some keywords function act as modifiers. There are two types of modifiers.
 
   In the above example the pattern 'index.php' is modified to inspect the HTTP uri buffer.
 
+
```

Comment:
I don't see this extra line changing the output appearance at all.  Is there another purpose to it?

Reply:
ok, I thought it would add whitespace between the items. 
No other purpose